package.json

```diff
@@ -58,14 +58,18 @@
     "builder:check": "npm run builder:lint"
   },
   "dependencies": {
-    "babel": "^5.8.21",
-    "babel-core": "^5.8.22",
-    "babel-loader": "^5.3.2",
+    "babel-cli": "^6.6.5",
+    "babel-core": "^6.6.4",
+    "babel-loader": "^6.2.4",
+    "babel-preset-es2015": "^6.6.0",
+    "babel-preset-es2015-webpack": "^6.4.0",
+    "babel-preset-react": "^6.5.0",
+    "babel-preset-stage-1": "^6.5.0",
     "rimraf": "^2.4.0",
-    "webpack": "^1.12.9"
+    "webpack": "^2.1.0-beta.4"
```

Comment:
The betas have been incubating since November of last year, and I've had great luck using it in `radium-grid`. Breaking changes are minimal and cosmetic. I'd give it a :+1: